siad/main.go

```diff
@@ -132,6 +132,9 @@ Explorer (e):
 
 // main establishes a set of commands and flags using the cobra package.
 func main() {
+	if build.DEBUG {
+		fmt.Println("Running with debugging enabled")
+	}
```

Comment:
On second consideration, I'm in favor of adding new fields to the `DaemonVersion` struct. Then we'd have the freedom to do this:
```go
type DaemonVersion struct {
    Version string
    Release string // dev, testing, standard
    Debug   bool
    // dynamically added at build time
    CommitSHA string
    BuildDate time.Time
}
```
This would help us diagnose remote machines because we could quickly identify exactly what commit was compiled, and when.